docs/perf/perf-notes.md

```
@@ -0,0 +1,57 @@
+# Performance Notes (`/zukan`)
+
+## Current Snapshot (2026-02-22)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)